web/server/codechecker_server/api/report_server.py

```diff
@@ -623,11 +623,25 @@ def get_report_details(session, report_ids):
         extended_data.filePath = file_path
         extended_data_list[report_id].append(extended_data)
 
+    # Get Comments for report data
+    comment_data_list = defaultdict(list)
+    comment_query = session.query(Comment, Report.id, File.filepath)\
```

Comment:
You are still querying the file path and joining the File table to this query which is unecessary. 
```suggestion
    comment_query = session.query(Comment, Report.id) \
```